src/styles/blocks/calendar.scss

```
@@ -0,0 +1,63 @@
+body,
+div {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  margin: 50px auto;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $calendar-width;
+  font-family: Arial, sans-serif;
+  font-size: $font-size;
+  color: black;
+
+  
+  &__day {
+    height: $cell-height;
+    width: $cell-width;
+    border: 1px solid black;
+    background-color: #eee;
+    flex-shrink: 0;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px) scale(110%);
+      animation-duration: 0.5s;
+    }
+  }
+
+  @for $n-day from 1 through 31 {
+    &__day:nth-of-type(#{$n-day})::before {
+      content: "#{$n-day}";
+      display: flex;
+      justify-content: center;
+      margin-top: ($cell-height / 2) - ($font-size / 2);
+    }
+  }
+
+  $days:
+    'monday' 1,
+    'tuesday' 2,
+    'wednesday' 3,
+    'thursday' 4,
+    'friday' 5,
+    'saturday' 6,
+    'sunday' 7;
+
+  @each $day, $index in $days {
+    &--#{$day} > &__day:first-of-type {
+      margin-left: ($index * $cell-width - $cell-width) + ($index * $gap - $gap);
```

```suggestion
      margin-left: ($index * $cell-width) + ($index * $gap);
```
начни days с нуля, и тогда не надо будет тут этих операций